packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -620,32 +625,17 @@ export class MarketOperationUtils {
         }
 
         // If RFQ liquidity is enabled, make a request to check RFQ liquidity
-        let comparisonPrice: BigNumber | undefined;
+        let comparisonPrice: ComparisonPrice | undefined;
```

Comment:
Looking at this in more detail, do we really need `comparisonPrice` at this point? or can we just keep `wholeOrderPrice`?

Reply:
```
                wholeOrderPrice = getComparisonPrices(optimizerResult, amount, marketSideLiquidity).
```

Reply:
done